UNRELEASED.md

```diff
@@ -12,6 +12,8 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Bug fixes
 
+- Fixed a bug with `type="number"` `TextField` where press-and-hold could increment or decrement infinitely ([#1029](https://github.com/Shopify/polaris-react/pull/1029))
```

Comment:
```suggestion
- Fixed `type="number"` `TextField` so press-and-hold doesn't increment or decrement infinitely ([#1029](https://github.com/Shopify/polaris-react/pull/1029))
```

https://github.com/Shopify/polaris-react/blob/master/documentation/Versioning%20and%20changelog.md#how-to-write-a-changelog-entry

Reply:
```suggestion
- Fixed `type="number"` `TextField` to prevent conditions where press-and-hold could increment or decrement infinitely ([#1029](https://github.com/Shopify/polaris-react/pull/1029))
```

Reply:
Looks great!